spec/griddler/email_spec.rb

```diff
@@ -289,6 +289,19 @@
     expect(body_from_email(text: body)).to eq 'Hello.'
   end
 
+  it 'should trim signature with non-breaking space after hyphens' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.